docs/src/about.md

```diff
@@ -110,7 +110,7 @@ Here we list the names of the maintainers of the `LazySets.jl` library, as well
 - Sebastián Guadalupe
 - Kostiantyn Potomkin
 - Frédéric Viry
-- Ueli Wechsler
+- [Ueli Wechsler](https://github.com/ueliwechsler), ETH Zurich
```

Comment:
I would not add the affiliation for only one person.

Reply:
i've opened https://github.com/JuliaReach/LazySets.jl/issues/1624 to remember adding affiliation to everyone

Reply:
Good, I added them here.